Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,15 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/archive/1.34.1.tar.gz"
+  version "1.34.1"
```

Comment:
I suspect this will be redundant. `brew` is capable of scanning relatively simple versions from URLs automatically :).